lib/foreman_rh_cloud/engine.rb

```diff
@@ -212,4 +212,15 @@ def self.register_scheduled_task(task_class, cronline)
   def self.with_local_advisor_engine?
     SETTINGS.dig(:foreman_rh_cloud, :use_local_advisor_engine) || false
   end
+
+  def self.ca_cert
+    # The reference to candlepin ca_cert_file can be removed
+    # once the setting is dropped. Foreman 3.15 introduces
+    # a single CA file that bundles all CAs.
+    if ::SETTINGS.dig(:katello, :candlepin, :ca_cert_file)
+      ::SETTINGS[:katello][:candlepin][:ca_cert_file]
+    else
+      ::SETTINGS[:ssl_ca_file]
+    end
```

Comment:
This `if` block is to future proof this chunk of code when we drop the Candlepin configuration settings in favor of the change to bundle CAs. Two references to this work:

https://github.com/theforeman/puppet-certs/pull/490
https://github.com/Katello/katello/pull/11327

Reply:
Given both PRs are merged, do we even need this if/else? Or are you trying to be compatible with pre 3.15 deployments here?

Reply:
Trying to be compatible. I'd rather merge this with compatibility and then follow up to remove it so there is a commit available with it.